projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -3640,11 +3925,23 @@ function render() {
 		return apply_filters( 'grunion_contact_form_field_html', $rendered_field, $field_label, ( in_the_loop() ? get_the_ID() : null ) );
 	}
 
+	// phpcs:disable WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
+
+	/**
+	 * Render the label.
```

Comment:
I've noticed when we say "render" we usually mean the function echos the output, but these are returning the HTML instead. Should we change these descriptions?

Reply:
fbd048da8694effa50a69985247381df31ba3fa5